docs/node-operators/snark-workers/getting-started.mdx

```diff
@@ -1,21 +1,87 @@
 ---
-title: Snark Workers
+title: Getting Started
```

Comment:
```suggestion
title: SNARK Workers Getting Started
```

Reply:
strongly recommend to keep the specific in the title to avoid ambiguous search results

Reply:
Moved to a SNARK workers sub-section, so each type of node has a section, an index describing it and a getting started page - for consistency and to make it easier to find.

Reply:
@halsaphi  I see the title as "Getting Started"

Did you mean to keep this title? or you want to accept the suggestion? or?

Reply:
hi @barriebyron - yes keep the title "Getting Started" - it's used in the other sub-sections 

so Node Operators has these sub sections
 Block Producer Node
SNARK workers - (I need to uppercase this)
Archive Nodes
Seed Peers
+ others

The idea is that for the node sections they have the "same" layout so it's easier to find the info you want ... i.e. getting started, connect to mainnet, connect to devnet etc

Reply:
I see what you mean about the search so I've changed the titles for each - Block Producers Getting Started, Archive Nodes Getting started etc

Reply:
This changes the title in the side bar - not sure it "looks" great but ...